types/acl/package.json

```
@@ -1,7 +1,7 @@
 {
     "private": true,
     "dependencies": {
-        "@types/mongodb": "^3.6.20",
-        "@types/redis": "^2.8.0"
+        "@types/redis": "^2.8.0",
+        "mongodb": "^4.4.1"
```

This was released today https://www.npmjs.com/package/mongodb/v/4.5.0
```suggestion
        "mongodb": "^4.5.0"
```